### PR TITLE
MNEMONIC-589: Document project releasing workflow

### PR DESCRIPTION
**Pull request by rhythmwind**
Signed-off-by: Zane <rhythmwind@hotmail.com>

### CODE REVIEW

File: src/develop/releaseflow.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+* 12.Vote on the release candidate.
+
+* 13.Check the latest candidate to Apache release repository as formal new version release.
+
+* 14.Perform formal release in git (see example below).
+    
+    ```
+    $ bin/release.sh bump 0.14.0 rc1 0.15.015.
+    ```
```
Comment (yzz127) [inline src/develop/releaseflow.md:60]: Here are some typos, could you please correct it?

**Review by yzz127 — CHANGES_REQUESTED** [review]
Fix typo, please git commit --amend and git push --force to update your local repo, so you don't create multiple commits to keep the commit history clean. Thanks!

**Comment by bigdata-memory** [comment]
It would be better if using --fixup instead of force-push in future PR updates, thanks.
https://fle.github.io/git-tip-keep-your-branch-clean-with-fixup-and-autosquash.html

**Comment by yzz127** [comment]
I am still seeing the following error, could you please verify the PR, thanks
tools/release.sh bump 0.14.0 rc1 0.15.015.

**Review by yzz127 — APPROVED** [review]
Change looks good, the commit history needs to get fixed

**Review by bigdata-memory — APPROVED** [review]
Great to me as well, thanks +1

**Pull request closed without merging**